visual-js/visual-snapshots/src/commands/pdf.ts

```diff
@@ -0,0 +1,33 @@
+import { Command } from "commander";
+import {
+  accessKeyOption,
+  branchOption,
+  buildIdOption,
+  buildNameOption,
+  customIdOption,
+  defaultBranchOption,
+  projectOption,
+  regionOption,
+  usernameOption,
+} from "./options.js";
+
+export const pdfCommand = () => {
+  return new Command()
+    .name("pdf")
+    .description("Create visual snapshots for each page of a PDF file")
+    .argument("<pdf-file-path>", "A path to a PDF file")
+    .addOption(usernameOption)
+    .addOption(accessKeyOption)
+    .addOption(regionOption)
+    .addOption(buildNameOption)
+    .addOption(branchOption)
+    .addOption(defaultBranchOption)
+    .addOption(projectOption)
+    .addOption(buildIdOption)
+    .addOption(customIdOption)
+    .action((pdfFilePath: string, options: Record<string, string>) => {
```

Comment:
As we know which options to expect, maybe it would be a good idea to define an interface with the options?

```ts
interface PdfCommandOptions {
  username: string;
  accessKey: string;
  customId?: string;
  // etc
}
```

and then annotate options as:
```ts
options: PdfCommandOptions
```

Reply:
I have already created an interface when playing with visual a little and have it stashed. I don't think this needs to be a part of this MR.